Rakefile

```diff
@@ -260,7 +260,7 @@ namespace :install do
 
     files.each do |f|
       path = File.expand_path("../tools/hooks/#{f}", __FILE__)
-      system "ln -s #{path} #{git_path}/#{f}"
+      RakeUtils.ln_s path, "#{git_path}/#{f}"
```

Comment:
This helper has an 'ensure' behavior, so it won't complain if the symlink is already correct.